src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -82,15 +83,19 @@ public static PackageSpec GetPackageSpec(Stream stream, string name, string pack
                 }
             }
 
+            packageSpec.Title = rawPackageSpec.GetValue<string>("title");
             packageSpec.Description = rawPackageSpec.GetValue<string>("description");
             packageSpec.Authors = authors == null ? new string[] { } : authors.ValueAsArray<string>();
             packageSpec.Owners = owners == null ? new string[] { } : owners.ValueAsArray<string>();
+            packageSpec.ContentFiles = contentFiles == null ? new string[] { } : contentFiles.ValueAsArray<string>();
```

Comment:
Does this work for content files? I would expect it to need more information to get it into the right tfm/langugage folder.